src/mame/drivers/hornet.cpp

```diff
@@ -224,6 +224,37 @@ f
     S/Scope      830A13  -       830A14  -
     S/Scope 2    -       -       -       -          (no ROMs, not used)
 
+    PWB405270B I/O board (Joystick pod board)
+    |------------------------------------------|
+    |     CN1         CN2                      |
+    |                                          |
+    |                              CN4         |
+    |                     DSW(8)               |
+    |                                          |
+    |058232                                    |
+    |                                          |
+    |                                          |
+    |                                          |
+    |                                  USB-B   |
+    |                                          |
+    |                                  USB-A   |
+    |    H8/3644                               |
+    |                                          |
+    |------------------------------------------|
+
+    This board normally comes packaged with Konami "Windy II" cabinets for JVS I/O support. Konami alternatively
+    packaged this board with NBA: Play by Play kits to support four players. This is a rare instance where a game
+    uses both JAMMA (players 1 and 2) AND JVS (players 3 and 4) inputs at the same time. A similar board, GO787-PWB(B)A,
+    was also developed for a "JAMMA to JVS converter" but uses the same H8/3644 MCU. Gradius 4 will always say "JVS Not Found"
+    if the main board (possibly from the PPC403GA's serial port) isn't communicating with its MCU even if the DIP 6 is set
+    to "off" position.
+
+	Notes:
+          058232 - Konami Custom Ceramic Package (SIL14)
+         H8/3644 - Hitachi H8/3644 Microcontroller. Unknown clock speed
```

Comment:
If you look carefully at the PCB photo you provided, you'll see "7.3728MHZ" silkscreened to the left of the H8 MCU near X1 (which it probably describes).

Reply:
Yea. Didn't notice till now. Fortunately, that value is already stored in "xtal.cpp". The JVS to JAMMA board using the same MCU also uses the same crystal and speed. Idk how to determine the multiplier though.

Reply:
It's usually the case with microcontrollers to have a direct XTAL hookup with any division being internal, unless the XTAL itself is used to clock other devices as well or if the value is out of spec (which it shouldn't be here).